xml/vm_security.xml

```diff
@@ -36,93 +36,17 @@
       confidential computing on the platform.
     </para>
   </sect1>
-  <sect1 xml:id="vm-security-enable-confidential-compute-module">
-    <title>Enabling confidential compute module</title>
-
-    <para>
-      The necessary packages for the AMD SEV-SNP feature are shipped via a confidential compute module.
-      You must enable it at system installation time or later via the SUSEConnect command-line
-      tool.
-    </para>
-
-    <itemizedlist>
-      <listitem>
-        <para>
-          To check whether the module is already enabled, run the command:
-        </para>
-<screen> &prompt.sudo; suseconnect -l</screen>
-        <para>
-          This displays the list of available modules with their activation status and commands to
-          enable the inactive modules.
-        </para>
-        <para>
-          The inactive confidential compute module appears as shown below:
-        </para>
-<screen>Confidential Computing Technical Preview Module 15 SP6 x86_64
-Activate with: suseconnect -p sle-module-confidential-computing/15.6/x86_64</screen>
-      </listitem>
-      <listitem>
-        <para>
-          To enable the confidential computing module technology preview, run the command:
-        </para>
-<screen> &prompt.sudo; <command>suseconnect -p sle-module-confidential-computing/15.6/x86_64</command>
-Registering system to SUSE Customer Center
-Updating system details on https://scc.suse.com ...
-Activating sle-module-confidential-computing 15.6 x86_64 ...
-Adding service to system ...
-Installing release package ...
-Successfully registered system</screen>
-        <para>
-          The confidential compute module is enabled and you can install the packages.
-        </para>
-      </listitem>
-    </itemizedlist>
-  </sect1>
   <sect1 xml:id="vm-security-verify-setup">
-    <title>Installing packages and setting up the base system</title>
-
-    <para>
-      The confidential compute module provides replacement packages supporting AMD SEV-SNP. To
-      ensure maximum compatibility, these packages are based on the code streams from
-      &productname; &productnumber;.
-    </para>
-
+    <title>Setting up the base system</title>
     <para>
-      The three components that need to be replaced are:
+      Load the MSR module, disable IOMMU and update the bootlaoder:
```

Comment:
I think we should include some of the info removed in the next paragraph in this intro paragraph. E.g. something like:

The &vmhost; requires minor configuration changes prior to running AMD-SNP enabled &vmguest;s. The default IOMMU configuration in SUSE Linux Enterprise Server 15 SP7 is passthrough mode. To use the AMD-SNP feature, the IOMMU must be configured in non-passthrough mode. This is required to prevent peripheral devices from writing to memory that belongs to an encrypted &vmguest; and destroying its data integrity. The msr kernel module is required to use the optional <command>snphost</command> tool.